torch/csrc/jit/serialization/export.h

```diff
@@ -188,5 +188,21 @@ TORCH_API void SetExportModuleMobileInfoConverter(
  */
 TORCH_API std::vector<std::string> export_opnames(const Module& m);
 
+struct TORCH_API BytecodeEmitDefaultInputsMode {
```

Comment:
```suggestion
struct TORCH_API BytecodeEmitDefaultValueForUnspecifiedArgMode {
```

Please also consider adding comment explaining what this guard is doing for future readers

Reply:
+1 What does having this mode armed or disarmed mean?

Reply:
@dhruvbird It's used to control if the TorchScript function emits the default value for unspecified argument or not.